indico/modules/events/abstracts/forms.py

```diff
@@ -277,6 +280,8 @@ def __init__(self, *args, **kwargs):
         elif not abstract.reviews:
             kwargs.setdefault('accepted_contrib_type', abstract.submitted_contrib_type)
         super().__init__(*args, **kwargs)
+        if self.event.cfa.force_track_selection:
+            self.accepted_track.validators.append(DataRequired())
```

Comment:
Not sure if this correctly works for the server-side validation or just for displaying the field as required.

Better to use our `inject_validators()` util before the super init call